form.css

```diff
@@ -29,4 +23,10 @@
     text-decoration: none;
     box-shadow: 0 2px 5px var(--light-gray);
   }
+
+  &.sm-full-width {
+    @media (--sm-viewport-down) {
+      width: 100%;
+    }
+  }
```

Comment:
I'd invert this to defaulting it to `100%` and then setting it to `auto` on larger resolutions. the down-based selectors seem relatively discouraged (and can be buggy).